pravega-sensor-collector/src/test/java/io/pravega/sensor/collector/file/rawfile/RawFileIngestServiceTest.java

```diff
@@ -0,0 +1,103 @@
+package io.pravega.sensor.collector.file.rawfile;
+
+import io.pravega.sensor.collector.DeviceDriverConfig;
+import io.pravega.sensor.collector.DeviceDriverManager;
+import io.pravega.sensor.collector.Parameters;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+public class RawFileIngestServiceTest {
```

Comment:
Looks like we can pull lots of common functionality into an abstract base class.
Both RawFileIngestServiceTest and ParquetFileIngestServiceTest can derive from it.